Sources/Danger/Danger.swift

```diff
@@ -26,6 +26,12 @@ private final class DangerRunner {
         logger.logInfo("Ran with: \(CommandLine.arguments.joined(separator: " "))")
         
         let cliLength = CommandLine.arguments.count
+        
+        guard cliLength - 2 > 0 else {
```

Comment:
Now is possible to run the code, but if you run it crashes

<img width="774" alt="screenshot 2018-11-05 at 20 01 02" src="https://user-images.githubusercontent.com/17830956/48023844-ce465700-e136-11e8-9a90-a68702dfec48.png">

This is why i added a nice message that replaces the crash